docs/source/_redirects.yml

```
@@ -0,0 +1,10 @@
+# Move "how-to" pages to the guides/ folder
+how-to-cache: guides/manage-cache
+how-to-discussions-and-pull-requests: guides/community
+how-to-downstream: guides/download
+how-to-inference: guides/inference
+how-to-manage: guides/repository
+how-to-model-cards: guides/modelcards
```

```suggestion
how-to-model-cards: guides/model-cards
```
i would maybe keep the dash here for consistency with other places we use this term

cc @meg-huggingface @EziOzoani

---

Good idea, I made the change.

---

updating to keep a naming consistency sounds good👍